bootstrap/config/kfctl_basic_auth.yaml

```diff
@@ -20,6 +20,7 @@ packages:
   - tf-serving
   - tf-training
 components:
+  - metacontroller
```

Comment:
can we keep it alphabetically ordered?

Reply:
ok, I see some error messages that might be related to this change - could you add the reason to description as Jeremy suggested?  thanks.